chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -246,7 +246,7 @@ def held_by_multipath?
         def fixed
           # This needs to be kept in sync with the number_of_drives method in
           # node_object.rb in the Crowbar framework.
-          @device =~ /^([hsv]d|dasd|cciss|xvd|nvme)/ && !removable && !cinder_volume
+          @device =~ /^([hsv]d|dasd|cciss|xvd|nvme)/ && !cinder_volume
```

Comment:
Do we really want this? Maybe it helps with the bug, but won't it bite us elsewhere?

Reply:
I feel uneasy about it, too. Perhaps we can check for this driver and zero the removable flag for them.

How will it bite us?

Reply:
I'm not sure it will, just saying that there was an intent in ignoring removable devices and we should investigate it.

Reply:
I think the main confusion would be with writeable media and USB sticks, though it is probably rare that those have unpartitioned space.